src/pyvism/runtime/builtins.py

```diff
@@ -6,7 +6,8 @@
 from io import StringIO
 from re import compile as re_compile
 import sys
-from typing import Any, Callable, Generic, TypeVar, TypeVarTuple
+from types import NoneType as UnsetType
```

Comment:
I understand your change, but my choice was deliberate.

Reply:
Alright

Reply:
I hope it's still fine that I used `UnsetType` instead of `type(None)` here:
```py
typing: list[type[MemoryValue]] = field(
	default_factory=lambda: [UnsetType] * MEMORY_MAX_ADDR
)
```

Reply:
> I understand your change, but my choice was deliberate.

Here is what I plan to change it to:
```py
class UnsetTypeMeta(type):
	def __instancecheck__(self, _: Any) -> Literal[True]:
		return True


class UnsetType(metaclass=UnsetTypeMeta):
	def __repr__(self) -> str:
		return "<Unset>"
```

Reply:
oh okay